scripts/read_GitHubcomment.py

```diff
@@ -0,0 +1,103 @@
+# This script takes the comment from GitHub and split it into two yaml files.
+# The first one is the new configuration yaml
+# The second one is the new subsets configuration file
+# Usage: python read_GitHubcomment.py --fileGitHub comment.tmp --fileSubset default_multi_subset.yaml
+
+# same code as read_GitHubcomment_5bis.py but create two function for managing config and subsets
```

Comment:
Should this comment line be removed? I don't see a `read_GitHubcomment_5bis.py` file